### PR TITLE
Live radio image placeholder for AMP

### PR DESCRIPTION
**Pull request by pjlee11**
Part of #4115

**Overall change:** Update the amp audio placeholder image and serve the image using the static image variable. 

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [x] This PR requires manual testing

### CODE REVIEW

**Comment by pjlee11** [comment]
**AMP image placeholder**
You can briefly see the grey placeholder image flash up before the iframe is loaded in

amp-image-placeholder

File: src/app/containers/ArticleMain/__snapshots__/index.test.jsx.snap [hunk 1]
```diff
@@ -208,7 +208,7 @@ exports[`ArticleMain should render a news article correctly 1`] = `
       rel="amphtml"
     />
     <link
-      href="/news/images/icons/icon-192x192.png"
+      href="http://localhost.bbc.com:7080/news/images/icons/icon-192x192.png"
```
Comment (pjlee11) [inline src/app/containers/ArticleMain/__snapshots__/index.test.jsx.snap:211]: These have changed as we are now have `SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN` and `SIMORGH_PUBLIC_STATIC_ASSETS_PATH` setup in Jest config - see `src/testHelpers/jest-setup.js`

**Review by ibMadbouly — APPROVED** [review]
🥇 LGTM

**Review by simonsinclair — APPROVED** [review]
This is a lot nicer than the previous placeholder. 👍 
I checked out the changes locally. The images are optimised. LGTM.

**Review by j-pendlebury — APPROVED** [review]

**Review by jroebu14 — APPROVED** [review]

**Pull request merged**